tests/extras/datasets/pandas/test_gbq_dataset.py

```diff
@@ -150,7 +149,7 @@ def test_save_load_data(self, gbq_dataset, dummy_dataframe, mocker):
             table_id, project_id=PROJECT, credentials=None, progress_bar=False
         )
         mocked_read_gbq.assert_called_once_with(
-            sql, project_id=PROJECT, credentials=None
+            project_id=PROJECT, credentials=None, **gbq_dataset._load_args
```

Comment:
```suggestion
            project_id=PROJECT, credentials=None, query=sql
```

to test that if we don't provide any specific `load_args` that we fallback to the default `query`